Source/Resources/Content/PowerShell/Exchange/ClientAccess/OwaMailboxPolicies/OwaMailboxPolicy-Default.json

```diff
@@ -152,6 +152,7 @@
   "defaultClientLanguage": 0,
   "defaultTheme": "",
   "delegateAccessEnabled": true,
+  "description":"What is it: Configures mailbox policies in Exchange Server for Outlook. \n Learn More:  https://docs.microsoft.com/en-us/exchange/clients/outlook-on-the-web/mailbox-policies?view=exchserver-2019",
```

Comment:
@metamenos , looks like a lot of stuff is changed for this one. See attached screenshots (or see Bytagig first deploy extensions) 
<img width="679" alt="Screen Shot 2020-08-24 at 11 25 21 AM" src="https://user-images.githubusercontent.com/70111688/91081878-a9b5ba80-e5fc-11ea-8b59-93257991fc49.png">
<img width="432" alt="Screen Shot 2020-08-24 at 11 25 31 AM" src="https://user-images.githubusercontent.com/70111688/91081888-ac181480-e5fc-11ea-961b-646b5d72486b.png">
<img width="360" alt="Screen Shot 2020-08-24 at 11 25 40 AM" src="https://user-images.githubusercontent.com/70111688/91081890-acb0ab00-e5fc-11ea-8040-6e3b18a79faf.png">
<img width="244" alt="Screen Shot 2020-08-24 at 11 25 48 AM" src="https://user-images.githubusercontent.com/70111688/91081892-ad494180-e5fc-11ea-92ef-e296046cc73f.png">
<img width="237" alt="Screen Shot 2020-08-24 at 11 25 59 AM" src="https://user-images.githubusercontent.com/70111688/91081894-ad494180-e5fc-11ea-8f08-2ef2020af1b8.png">
<img width="489" alt="Screen Shot 2020-08-24 at 11 26 15 AM" src="https://user-images.githubusercontent.com/70111688/91081895-ade1d800-e5fc-11ea-8303-0ab35fce77d7.png">

Reply:
**What does this do?**
Configures the default settings for Outlook on the web. The baseline uses the default configurations provided by Microsoft. 

**Why should you use this?**
If you want to use the default Outlook on the web policies.

**What is the end-user impact?**
N/A <-- @metamenos , anything you can think of? I said N/A because we're using the default 

**Learn more**
[OwaMailboxPolicy](https://docs.microsoft.com/en-us/powershell/module/exchange/set-owamailboxpolicy?view=exchange-ps)